packages/nimble-components/src/utilities/unit-format/default-unit-format.ts

```diff
@@ -3,7 +3,7 @@ import type { ScaledUnitFormat } from './scaled-unit-format/scaled-unit-format';
 import type { UnitScale } from './unit-scale/unit-scale';
 import { passthroughUnitScale } from './unit-scale/passthrough-unit-scale';
 
-type NumberStyle = 'default' | 'leadingZero' | 'exponential';
+type SignDisplay = Intl.NumberFormatOptions['signDisplay'];
```

Comment:
can you do something like:

```ts
// workaround comment
const signDisplay = 'negative' as Intl.NumberFormatOptions['signDisplay'];
```

and use that constant instead in the spots below?